bundles/org.dataflowanalysis.analysis.dfd/src/org/dataflowanalysis/analysis/dfd/core/DFDPartialFlowGraphFinder.java

```diff
@@ -0,0 +1,125 @@
+package org.dataflowanalysis.analysis.dfd.core;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import org.dataflowanalysis.analysis.core.AbstractPartialFlowGraph;
+import org.dataflowanalysis.dfd.datadictionary.AbstractAssignment;
+import org.dataflowanalysis.dfd.datadictionary.DataDictionary;
+import org.dataflowanalysis.dfd.datadictionary.Pin;
+import org.dataflowanalysis.dfd.dataflowdiagram.DataFlowDiagram;
+import org.dataflowanalysis.dfd.dataflowdiagram.Flow;
+import org.dataflowanalysis.dfd.dataflowdiagram.Node;
+
+/**
+ * The DFDPartialFlowGraphFinder determines all partial flow graphs contained in a model
+ */
+public class DFDPartialFlowGraphFinder {
```

Comment:
The PCMGraphFinder is instantiated, this one only provides static methods. Is there a good reason for this? Otherwise, this should be unified (an probably not use static methods)

Reply:
Moved into non-static class in 67f4dbbc90ca421b0c9cc6d168acf8592e3c66fa